pkg/sdk/poc/README.md

```diff
@@ -35,17 +35,71 @@ make clean-generator-poc run-generator-poc
+   - right now, we have `valueSet` function that doesn't take into consideration edge cases, e.g. with slice where sometimes
+   we would like to do something like `alter x set y = ()` (set empty array to unset `y`). Those edge cases have cause on our
+   validation, and it determines sometimes if we'll return an error or not, which can lead to bugs!
+- refactor generation of `Describe`, so it will tak context and request as arguments
+  - all the interface functions should have context and request as arguments for the sake of API consistency and generation simplicity
+- split templates into multiple templates (e.g. ImplementationTemplate) to improve readability
+  - example implementation - https://go.dev/play/p/Cgt0sISlzwK
+  - divide implementation templates for Show, Describe and others
+- check if SelfIdentifier implementation is correct (mostly type, because it's derived from interface obj) by checking
+if there's a resource with different types of identifiers across queries (e.g. Create <AccountObjectIdentifier>, Alter <SchemaObjectIdentifier>) 
+- we should specify prefix / postfix standard for top-level items in _def.go files to avoid any conflicts in the package
+- remove name argument from QueryStruct in the Operation, because Opt structs in the Operation will have name from op name + interface field and not query struct itself
+- Derive field name from QueryStruct, e.g. see network_policies_def where we can remove "Set" field, but we have to make a convention of creating nested struct with
+name pattern like <interface name><name> e.g. NetworkPoliciesSet or NetworkPolicySet, then we could automatically remove prefix and we'll name field with postfix, so "Set" in this case
 
 ##### Improvements
 - automatic names of nested `struct`s (e.g. `DatabaseRoleRename`)
 - check if generating with package name + invoking format removes unnecessary qualifier
 - consider merging templates `StructTemplate` and `OptionsTemplate` (requires moving Doc to Field)
-- add unit tests to this generator
+- expand unit tests generation
+- experiment with Snowflake table (any table) representation in Go in order to implement DbStruct -> PlainStruct convert function
+  - see if *string can have similar effect as sql.NullString (check go-snowflake connector ?)
+     - if yes, then we should be using pointers instead of abstractions like sql.NullString and we can
+     modify ShowMapping and DescribeMapping to generate convert function with automatic conversion (as we have in DTOs).
+     warehouses.go is a good place to start with when planning mapping strategy, because there's a lot of different mapping cases.
+- when calling .SelfIdentifier we can implicitly also add validateObjectIdentifier validation rule
+- enforce user to use KindOf... functions with interface
+  - example implementation - StringTyper implements Typer and all the KindOf... functions use StringTyper to return Typer easily - https://go.dev/play/p/TZZgSkkHw_M
```

Comment:
please add here a point about keeping track of generated structs, because the implementation now is not the prettiest one